magento/ece-tools
# MAGECLOUD-2907: [Cloud Docker] Verify sendmail functionality

### Description


### Fixed Issues (if relevant)

1. https://magento2.atlassian.net/browse/MAGECLOUD-2907

### Manual testing scenarios

* Add `127.0.0.1 magento2.docker` to `/etc/hosts`
* Pull latest images `make pull`
* Generate config `./vendor/bin/ece-tools docker:config:convert`
* Change docker/gobal.env to `ENABLE_SENDMAIL=true`
* Redeploy env with `make up`
* Open frontend `magento2.docker` and sign-up as a customer with a real e-mail (don't use `@adobe.com`, for me `@gmail.com` works fine)
* Check inbox for the email

### Contribution checklist
